File: cakk-domain/mysql/src/main/java/com/cakk/domain/mysql/dto/param/shop/ShopOperationParam.java
package com.cakk.domain.mysql.dto.param.shop;

import java.time.LocalTime;

import com.cakk.common.enums.Days;

public record ShopOperationParam(
	Long cakeShopId,
	Days operationDay,
	LocalTime operationStartTime,
	LocalTime operationEndTime
) {
}

File: cakk-domain/mysql/src/main/java/com/cakk/domain/mysql/dto/param/shop/ShopOperationParam.kt
package com.cakk.domain.mysql.dto.param.shop

import com.cakk.common.enums.Days
import java.time.LocalTime

data class ShopOperationParam(
	val cakeShopId: Long,
	val operationDay: Days,
	val operationStartTime: LocalTime,
	val operationEndTime: LocalTime
)
